pkg/deploy/controller/deploymentconfig/deploymentconfig_controller.go

```diff
@@ -125,6 +148,15 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 				if err != nil {
 					return err
 				}
+				// We need to make sure we own that RC or adopt it if possible
+				isOurs, err := cm.ClaimReplicationController(rc)
+				if err != nil {
+					return fmt.Errorf("DeploymentConfigController failed to claim the RC: %s", err)
```

Comment:
nit: `error while deploymentConfigController claiming the RC %s/%s: %v` (add the rc name)